README.md

```diff
@@ -33,32 +37,31 @@ resources:
     params:
       driver: rabbitmq
       datasource: amqp://guest:guest@localhost:5672
-      
 ```
 
-[List of available resources](http://alileza.github.io/tomato/resources)
-
-Once you're ready with all your resources you needed, you're good to start writing your features. You can find some of examples [here](https://github.com/alileza/tomato/tree/0.1.0/examples/features)
+Once your tomato.yml is set up with all the resources your application requires, you can begin writing features. Features are pre-defined behavior tests that are run against your resources to determine if the application performs as expected. You can find some examples [here](https://github.com/alileza/tomato/tree/0.1.0/examples/features)
 
 `example.feature`
 ```gherkin
-Feature: behavior test example
-
-  Scenario: Delete customer on DELETE http request
-    Given listen message from "my-awesome-queue" target "customers:deleted"
-    Given set "my-awesome-postgres-db" table "customers" list of content
-        | name    | country |
-        | cembri  | id      |
-    Given "httpcli" send request to "DELETE /api/v1/customers" with body
+Feature: Customer removal via an HTTP API request
+
+  # Set the stage for the test
+  Background:
+    Given the table "customers" in "my-awesome-postgres-db" with content
+      | name    | country |
+      | cembri  | id      |
+
+  Scenario: Remove cembri when a rabbitmq delete message is received
+    Given a message received from "my-awesome-queue" target "customers:deleted"
+    When the "httpcli" sends a DELETE request to "/api/v1/customers" with body
         """
             {
                 "name":"cembri"
             }
         """
-    Then "my-awesome-postgres-db" table "customers" should look like
-        | customer_id | name    |
-    Then message from "my-awesome-queue" target "customers:deleted" count should be 1
-    Then message from "my-awesome-queue" target "customers:deleted" should look like
+    Then the table "customers" in "my-awesome-postgres-db" should not have a "name" cembri
```

Comment:
is this a new function?

Reply:
I was thinking of that being a new function. It would just do a `select * from customers where name = "cembri"`. I think maintaining table schemas could be kind of annoying, this way it just checks if row is there with that value (could extend to count of rows with those values, and things like that too)

Reply:
well then we might need to add functionality first, so the docs wouldn't be misleading. Also, I think using keyword `contains` in this case would be more correct? like

```
Then the table "customers" in "my-awesome-postgres-db" should NOT contains a row with "name = cembri"
Then the table "customers" in "my-awesome-postgres-db" should NOT contains a rows
| name   | age |
| cembri | 25 |
```

then we can also have the opposite function `should contain`